proxy/coco/src/state.rs

```diff
@@ -39,6 +39,8 @@ pub use error::Error;
 pub struct State {
     /// Internal handle on [`PeerApi`].
     pub(crate) api: PeerApi<keys::SecretKey>,
+    /// Signer to sign artifacts generated by the user.
+    pub signer: signer::BoxedSigner,
```

Comment:
Should the signer be accessible publically? :eyes:

Reply:
No, it does not need to be public. Will fix